wiki/dfg_nodes.md

```diff
@@ -0,0 +1,53 @@
+# Understanding Data Flow Graph (DFG) Node Labels in Flowr
+
+## Overview
+The Data Flow Graph (DFG) in Flowr represents the dependencies between different code components. Each node in the DFG contains specific labels that provide information about the operation being performed, the code span, and its relationship with other nodes.
+
+This document clarifies the meaning of these labels and how to interpret them.
+
+## Node Label Structure
```

Comment:
This node label structure is _only_ part of the mermaid serialization of the [DFG](https://github.com/flowr-analysis/flowr/wiki/Dataflow-Graph) and not by any means a global/given thing.

Reply:
See:

https://github.com/flowr-analysis/flowr/blob/7e7fd221dc521371cdae2dc08e009ffbbd749070/src/util/mermaid/dfg.ts#L247-L260